src/cli.ts

```diff
@@ -67,12 +68,16 @@ if (require.main === module) {
   const args = commander
     .name('nativefier')
     .version(packageJson.version, '-v, --version')
-    .arguments('<targetUrl> [dest]')
+    .arguments('[targetUrl] [dest]')
     .action((url, outputDirectory) => {
       positionalOptions.targetUrl = url;
       positionalOptions.out = outputDirectory;
     })
     .option('-n, --name <value>', 'app name')
+    .option(
+      '--upgrade <value>',
+      'Perform an upgrade of an app created with an older version of Nativefier. Provide the full path of the application executable.',
```

Comment:
Maximum conciseness for `--help` options, pls:

```suggestion
      'Upgrade an app built by an older Nativefier. You must pass the full path to the existing app executable.',
```